internal/util/debug/debug.go

```diff
@@ -19,35 +19,49 @@ import (
 	"bytes"
 	"context"
 	_ "expvar" // for metrics
+	"fmt"
+	"log"
+	"math/rand"
 	"net"
 	"net/http"
 	_ "net/http/pprof" // for profiling
 	"text/template"
 	"time"
 
+	"github.com/FerretDB/FerretDB/internal/util/must"
 	"github.com/arl/statsviz"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
+	io_prometheus_client "github.com/prometheus/client_model/go"
 	"go.uber.org/zap"
-
-	"github.com/FerretDB/FerretDB/internal/util/must"
 )
 
 // RunHandler runs debug handler.
 func RunHandler(ctx context.Context, addr string, r prometheus.Registerer, l *zap.Logger) {
 	stdL := must.NotFail(zap.NewStdLogAt(l, zap.WarnLevel))
 
-	http.Handle("/debug/metrics", promhttp.InstrumentMetricHandler(
+	metricHandler := promhttp.InstrumentMetricHandler(
 		r, promhttp.HandlerFor(prometheus.DefaultGatherer, promhttp.HandlerOpts{
 			ErrorLog:          stdL,
 			ErrorHandling:     promhttp.ContinueOnError,
 			Registry:          r,
 			EnableOpenMetrics: true,
 		}),
-	))
+	)
+	prometheus_metrics, err := prometheus.DefaultGatherer.Gather()
+	fmt.Printf("prometheus_metrics: %v\n", prometheus_metrics)
+	if err != nil {
+		log.Fatalf("error in Gathering prometheus metrics: %v", err)
+	}
+
+	http.Handle("/debug/metrics", metricHandler)
 
 	opts := []statsviz.Option{
 		statsviz.Root("/debug/graphs"),
+		statsviz.TimeseriesPlot(ferretdb_postgresql_metadata_databases_barPlot()),
+		statsviz.TimeseriesPlot(ferretdb_postgresql_pool_size_barPlot()),
+		statsviz.TimeseriesPlot(promhttp_metric_handler_requests_total_stackPlot()),
+		statsviz.TimeseriesPlot(promhttp_metric_handler_requests_in_flight_scatterPlot()),
```

Comment:
Instead of adding graphs for those 4 metrics, let's generate graphs for all metrics

Reply:
Yes let's do that

Reply:
Generate _programmatically_, not write a function for each metric. We often add and remove metrics, all of them are available in `prometheus.DefaultGatherer`, we could generate graphs on a fly

Reply:
My apologies. I'll make the Changes :+1: